pkg/security/ebpf/manager.go

```diff
@@ -26,7 +26,7 @@ func NewDefaultOptions() manager.Options {
 		DefaultKProbeMaxActive: 512,
 
 		// DefaultPerfRingBufferSize is the default buffer size of the perf buffers
-		DefaultPerfRingBufferSize: 128 * os.Getpagesize(),
+		DefaultPerfRingBufferSize: 4096 * os.Getpagesize(),
```

Comment:
Should this be made configurable?

Reply:
This parameter is a bit sensitive and will have an impact on our detection capabilities. Right now, we don't need to make it configurable, but we might in the future.